### PR TITLE
Sets configuration

### PR DESCRIPTION
**Pull request by jcchavezs**
Ping @codefromthecrypt

### CODE REVIEW

File: e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+	err = e2e.Run(e2e.Config{
+		ProxiedEntrypoint: ts.URL,
+		HttpbinEntrypoint: ts.URL,
+	})
+	require.NoError(t, err)
+}
+
+// testLogger is a api.Logger implementation for testing purposes.
+type testLogger struct{ t *testing.T }
```
Comment (jcchavezs) [inline e2e_test.go:86]: This is something I use a lot every time. Worth to include in http-wasm sdk? @codefromthecrypt

File: e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+var _ api.Logger = testLogger{}
+
+func (l testLogger) IsEnabled(_ api.LogLevel) bool { return l.t != nil }
+
+var levels = map[api.LogLevel]string{
```
Comment (jcchavezs) [inline e2e_test.go:92]: Shall we add this to http-wasm go sdk @codefromthecrypt?

**Comment by anuraaga** [comment]
Sorry I don't think I can put time to review PRs in this repo. I would personally recommend moving it to either a personal repo or tetratelabs, http-wasm doesn't seem to be in coraza's priorities to be honest.

**Comment by anuraaga** [comment]
Another suggestion is to focus on a proxy-wasm wrapper implementation for http-wasm. Fragmentation is bad, we've already gone through it with observability. I think for coraza, the best situation for now is to focus on coraza-proxy-wasm - if that module can be magically loaded in http-wasm then no problem. If not, it probably needs to be able the with abi tweaks to support it since interoperability is king in middleware.

**Comment by jcchavezs** [comment]
Thanks @anuraaga. I am moving this repo to my personal namespace as I
opened this but it did not went through any consensus on to support it.

**Pull request merged**